app/controllers/sp_components_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
 
   def new
     @component = NewSpComponentEvent.new
+    @hub_environments = Rails.configuration.hub_environments.keys
```

Comment:
This was looking very good until, I tried running the app. I got this error

```undefined method `hub_environments' for #<Rails::Application::Configuration:0x00007f86f24386d8>```

Reply:
I think this would fine after #125

Reply:
Yeah I gotta wait for 125 to be merged then rebase, then this will be ok :)